python/pyspark/sql/tests.py

```diff
@@ -511,7 +517,6 @@ def test_udf_with_order_by_and_limit(self):
         my_copy = udf(lambda x: x, IntegerType())
         df = self.spark.range(10).orderBy("id")
         res = df.select(df.id, my_copy(df.id).alias("copy")).limit(1)
-        res.explain(True)
```

Comment:
Can we remove this?
I guess it's only the print out of the plan, but I'd revert it just in case because it's not related to this pr.

Reply:
Yes, I think this is removed.